source/windows.js

```diff
@@ -41,26 +41,18 @@ const mIsExe = async (file, cwd, PATH) => {
 		// `PATH` parts can be double quoted on Windows
 		.map(part => part.replace(/^"(.*)"$/, '$1'));
 
-	// For performance, parallelize and stop iteration as soon as an *.exe of *.com file is found
+	// For performance, parallelize and stop iteration as soon as an *.exe or *.com file is found
 	try {
-		await Promise.all(exeExtensions
-			.flatMap(extension =>
-				[cwd, ...parts].map(part => `${path.resolve(part, file)}${extension}`))
-			.map(async possibleFile => {
-				try {
-					await stat(possibleFile);
-				} catch {
-					return;
-				}
-
-				// eslint-disable-next-line no-throw-literal
-				throw 0;
-			}));
+		await Promise.any(
+			exeExtensions.flatMap(extension => [cwd, ...parts]
+				.map(part => access(`${path.resolve(part, file)}${extension}`)),
```

Comment:
Should we also do this change? Feels better to understand to me.

```suggestion
				.map(part => access(path.join(part, file + extension)),
```

Reply:
This would change the result, when `file` is an absolute file path.

```
> path.win32.resolve('C:\\a\b\c', 'C:\\d')
'C:\\d'
> path.win32.join('C:\\a\b\c', 'C:\\d')
'C:\\a\bc\\C:\\d'
```

Reply:
Thanks!